src/main/resources/templates/db_scripts/renameModesOfIssuance.sql

```diff
@@ -0,0 +1,23 @@
+START TRANSACTION;
+
+UPDATE ${myuniversity}_${mymodule}.mode_of_issuance AS it
+SET jsonb = jsonb_set(it.jsonb, '{name}', '"integrating resource"')
+WHERE it.jsonb->>'name' = 'Integrating Resource';
```

Comment:
Can we use id here instead?

Reply:
Maybe both, `id` and `name` to prevent value overwriting as described @marcjohnson-kint  in comment above.

Reply:
Based upon [Charlotte's comment](https://issues.folio.org/browse/MODINVSTOR-431?focusedCommentId=70032&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-70032) on the story, this work should include a data upgrade script that always overwrites the name of these records, whether they have changed or not. Meaning we should only match on ID.

Reply:
Changed